src/JsonSchema/index.js

```diff
@@ -9,30 +9,30 @@
  * OF ANY KIND, either express or implied. See the License for the schemaific language
  * governing permissions and limitations under the License.
  */
-
-import React, { Fragment } from 'react'
-import { Flex, Heading, View, Well } from '@adobe/react-spectrum'
-import { InlineCode } from '../InlineCode'
-import { Table, TBody, Tr, Td } from '../Table'
+import React from 'react'
+import { Heading } from '@adobe/react-spectrum'
+import { Item, TabList, TabPanels, Tabs } from '@react-spectrum/tabs'
+import { JsonSchemaView, JsonSchemaProperty } from './JsonSchemaView'
+import { JsonSchemaValidate } from './JsonSchemaValidate'
+import { JsonSchemaRaw } from './JsonSchemaRaw'
+import { JsonSchemaInfo } from './JsonSchemaInfo'
 
 export const JsonSchema = ({ schema = {}, ...props }) => {
   const {
     $id,
     _id,
-    id,
-    slug,
```

Comment:
If we remove these two lines then when Gatsby tries to create a page from a JSON Schema it will pass in the `id` and `slug` properties and they'll end up in `remainingProps`. That means they'll get output in lines 37-43. I put them in here so the component would ignore them.